src/components/Onboarding/utils/progress-utils.ts

```diff
@@ -22,7 +22,10 @@ export const setItem = (account, chainId, item) => {
 
 const GARDEN_ASSETS = ['logo_type', 'logo', 'token_logo']
 
-function recoverBlob(file, type = 'dataUrl') {
+function recoverBlob(
+  file?: { content: any; blob: { path: any } },
+  type = 'dataUrl'
+) {
```

Comment:
Its possible extract to named type here or get better type already used someelse.